cli/vm/cli_test.go

```diff
@@ -684,6 +684,16 @@ func TestLoad_RunWithCALLT(t *testing.T) {
 		)
 		e.checkNextLine(t, "READY: loaded \\d* instructions")
 		e.checkStack(t) // Nothing on stack, successful execution.
+
+		// Exit the VM prompt.
+		finalizeInteropContext(e.cli.shell)
+		l := getReadlineInstanceFromContext(e.cli.shell)
+		require.NotNil(t, l)
+		err := l.Close()
+		require.NoError(t, err)
+		exit := getExitFuncFromContext(e.cli.shell)
+		require.NotNil(t, exit, "Exit function should not be nil")
+		exit(0)
```

Comment:
This code is already in the VM, so please, follow DRY and don't repeat code if it's possible. Here you just need to call `exit` VM command as the last command of the test. We already had problems like #3154, see the #2736 and #2737, so before you try to fix some bug it's a good practice to search for the existing similar issues:
https://github.com/nspcc-dev/neo-go/blob/00a1a8da56e1149ee9d563b96f3d8d09e112c3ac/cli/vm/cli_test.go#L1069-L1072

Reply:
The same thing should be done for all state-dependent users of testing VM.